src/lib/y2storage/volume_specification.rb

```diff
@@ -236,6 +254,20 @@ def fs_type_configurable?
       fs_types.size > 1
     end
 
+    # @see #reformat
+    #
+    # @return [Boolean]
+    def reformat?
+      reformat
```

Comment:
NP: Why not an alias method?

Reply:
No strong reason, I copied it from `Planned::CanBeFormatted`, I can change it.

Reply:
Fixed anyways.